CMP1903_A1_2324/Program.cs

```diff
@@ -8,12 +8,32 @@ namespace CMP1903_A1_2324
 {
     internal class Program
     {
+
         static void Main(string[] args)
         {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
+
+            while (true)
+            {
+                Console.Write("Would you like to roll the dice? (Y or N)");
+                string reRoll = Console.ReadLine();
```

Comment:
A comment explaining this line would be useful

Reply:
Done 👍🏻